### DISCUSSION TITLE
curl ssl certificate problem on download quick-lint-js

### DISCUSSION THREAD

**Discussion by HUAHUAI23** [mason-org/mason.nvim#310] [Q&A]
![image](https://user-images.githubusercontent.com/43649186/185309434-501af02e-35ae-4b5d-9048-30aa6b007a53.png)
![image](https://user-images.githubusercontent.com/43649186/185309460-6d488eb9-073e-4dbe-97fc-7c3ea1ced496.png)

https://c.quick-lint-js.com/releases/2.8.0/manual/linux.tar.gz

**Comment by williamboman** [comment]
Hello! Hm, seems to be a certificate issue on that domain. I'd recommend reporting it upstream
